draft-ietf-cbor-update-8610-grammar.md

```diff
@@ -367,17 +368,27 @@ Some specifications operate on ranges of tag numbers, e.g., {{?RFC9277}}
 has a range of tag numbers 1668546817 (0x63740101) to 1668612095
 (0x6374FFFF) to tag specific content formats.
 This can currently not be expressed in CDDL.
+Similar considerations apply to simple values (`#7.`xx).
 
-This update extends this to:
+This update extends the syntax to:
 
 ~~~ abnf
 ; new rules collectively defining the tagged case:
-type2 =/ "#" "6" ["." tag-number] "(" S type S ")"
-tag-number = uint / ("<" type ">")
+type2 =/ "#" "6" ["." head-number] "(" S type S ")"
+       / "#" "7" ["." head-number]
+head-number = uint / ("<" type ">")
```

Comment:
Does this need to include `uint` even? There is already a line in the type2 that is being extended that says ` / "#" DIGIT ["." uint]  ; major/ai` -- wouldn't that match first anyway, and thus no uint ever comes "down" this far?

(Not new for `#7`)

Reply:
The sequence in the collected ABNF in Appendix A is

``` abnf
      / "#" "6" ["." head-number] "(" S type S ")"
      / "#" "7" ["." head-number]
      / "#" DIGIT ["." uint]                ; major/ai
```

So, in a PEG parser, the second line wins over the third line.
In a traditional parser generator, this might be an ambiguity; ABNF does not require avoiding ambiguities though.